torchmdnet/module.py

```diff
@@ -163,15 +163,21 @@ def step(self, batch, loss_fn_list, stage):
             batch.y = batch.y.unsqueeze(1)
         for loss_fn in loss_fn_list:
             step_losses = self._compute_losses(y, neg_dy, batch, loss_fn, stage)
+
+            loss_name = loss_fn.__name__
+
+            if self.hparams.neg_dy_weight > 0:
+                self.losses[stage]["neg_dy"][loss_name].append(
+                    step_losses["neg_dy"].detach()
+                )
```

Comment:
Is it ok to append only when it is non zero? As opposed to append zero

Reply:
Essentially you don't care about the y loss metrics if `y_weight` is 0. If you don't append to the losses, then you are not going to have useless metrics in the logger (wandb) and in the `results_dict`.

Reply:
Please remove empty lines